packages/wrangler/src/cloudchamber/images/list.ts

```diff
@@ -144,15 +152,19 @@ async function ListTags(
 	if (json) {
 		logger.log(JSON.stringify(responses, null, 2));
 	} else {
-		const rows = responses
-			.map((resp) => {
-				return {
-					REPOSITORY: resp.name,
-					TAG: resp.tags.join(" "),
-				};
-			})
-			.flat();
-		logger.table(rows);
+		const rows = responses.map((r) => r.tags.map((t) => [r.name, t])).flat();
+		const headers = ["REPOSITORY", "TAG"];
+		const widths = new Array(headers.length).fill(0);
+
+		// Find the maximum length of each column (except for the last)
+		for (let i = 0; i < widths.length - 1; i++) {
+			widths[i] = rows.map((r) => r[i].length).reduce((a, b) => Math.max(a, b), headers[i].length);
```

Comment:
Is this equivalent to:
```suggestion
			widths[i] = Math.max(...rows.map((r) => r[i].length));
```